lib/__tests__/_tokenUtils.test.ts

```diff
@@ -104,4 +120,28 @@ describe("tokenUtils", () => {
       });
     });
   });
+
+  describe("getAuthenticatedUserToken", () => {
+    it("should return undefined if not set", () => {
+      const authenticatedUserToken =
+        analyticsInstance.getAuthenticatedUserToken();
+      expect(authenticatedUserToken).toBeUndefined();
+    });
+    it("should return current authenticatedUserToken", () => {
+      analyticsInstance.setAuthenticatedUserToken("008");
+      const authenticatedUserToken =
+        analyticsInstance.getAuthenticatedUserToken();
+      expect(authenticatedUserToken).toEqual("008");
+    });
+    it("should accept a callback", () => {
+      analyticsInstance.setAuthenticatedUserToken("009");
```

Comment:
If the callback is never called this test will still pass. You need to add something like `expect.assertions(2);` to ensure that the expected assertions are made.